unittests/CppInterOp/InterpreterTest.cpp

```diff
@@ -103,3 +107,19 @@ TEST(InterpreterTest, DetectSystemCompilerIncludePaths) {
   Cpp::DetectSystemCompilerIncludePaths(includes);
   EXPECT_FALSE(includes.empty());
 }
+
+TEST(InterpreterTest, CodeCompletion) {
+#if CLANG_VERSION_MAJOR >= 18 || USE_CLING
+  std::vector<std::string> cc;
+  Cpp::Declare("int foo = 12;");
+  Cpp::CodeComplete("f", cc);
+  size_t cnt = 0;
+  for (auto& r : cc) {
```

Comment:
Maybe we should explain why we filter these candidates here. I suspect that's because of cling. In that case we can probably surround it with `#ifdef USE_CLING`.